src-electron/generator/helper-endpointconfig.js

```diff
@@ -100,11 +100,15 @@ function endpoint_fixed_profile_id_array(options) {
  * @returns C array including the { } brackets
  */
 function endpoint_fixed_parent_id_array(options) {
-  return (
-    '{ ' +
-    this.endpoints.map((ep) => ep.parentEndpointIdentifier).join(', ') +
-    ' }'
-  )
+  let parentIds = []
+  this.endpoints.forEach((ep) => {
+    if (ep.parentEndpointIdentifier == null) {
+      parentIds.push('NULL')
```

Comment:
You are hard-coding C `NULL` here. Not the biggest evil, but I thought we have this externalized somewhere?